### PR TITLE
Localize test.md to Gujarati.

### PR DESCRIPTION
**Pull request by devarsh10**
Signed-off-by: Devarsh Shah [devarshshah2608@gmail.com](devarshshah2608@gmail.com)
Localizes ``content/en/docs/test.md`` to Gujarati
This is an intial PR as may need some improvements!
Tracking issue: #38740

### CODE REVIEW

**Comment by sftim** [comment]
Thanks

We don't yet have an official Gujurati localization
/hold

File: content/gu/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: रोलिंग अपडेट करना 
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="hi">
```
Comment (sftim) [inline content/gu/docs/tutorials/kubernetes-basics/update/update-intro.html:8]: _Hindi?_

Comment (devarsh10) [reply content/gu/docs/tutorials/kubernetes-basics/update/update-intro.html:8]: Yes it's localized in hindi.

Comment (sftim) [reply content/gu/docs/tutorials/kubernetes-basics/update/update-intro.html:8]: Shouldn't it be Gujurati? That's what I'd expect to see.

Comment (devarsh10) [reply content/gu/docs/tutorials/kubernetes-basics/update/update-intro.html:8]: Oh..okay okay..
I misunderstood.
Let me localized it in Gujarati.

**Comment by divya-mohan0209** [comment]
Hello @devarsh10 , gentle reminder.

Are the efforts to localise Gujarati still ongoing? If there are no commits made, please note that this PR will rot & close itself after a week.

**Comment by devarsh10** [comment]
Okay @divya-mohan0209.
Working on it.

**Pull request closed without merging**